coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
In the future we should make sure to add a line that says `/coverage/` to our `.gitignore` file to make sure that these files don't get saved into Git.